android/java/res/layout/fragment_brave_location_bar_qr_dialog.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) 2022 The Brave Authors. All rights reserved.
+     This Source Code Form is subject to the terms of the Mozilla Public
+     License, v. 2.0. If a copy of the MPL was not distributed with this file,
+     You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+<LinearLayout
```

Comment:
can we use coordianate layout to avoid nested layouts ?

Reply:
sure but it won't help much to avoid nested layouts as we have to add scrollview to support landscape mode in phone.